website/src/app/[lang]/[region]/(website)/newsletter/archive/[slug]/page.tsx

```
@@ -0,0 +1,42 @@
+	const htmlRaw = getNewsletterHTML(slug);
+	if (!htmlRaw) notFound();
+
+	const html = sanitizeNewsletterHtml(htmlRaw);
+	if (!html) notFound();
+
+	return (
+		<div className="mx-auto max-w-4xl p-4" suppressHydrationWarning>
+			<Link href="/newsletter/archive" className={linkCn({ underline: 'hover' })}>
+				← {translator.t('archive.title')}
+			</Link>
+			<div className="mt-8" dangerouslySetInnerHTML={{ __html: html }} />
```

Biome (2.1.2)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)